tests/integration/api/controllers/users.spec.js

```diff
@@ -1605,6 +1608,42 @@ describe('Users API', () => {
     });
   });
 
+  describe('GET api/v2/users/{username}', () => {
```

Comment:
Could you add a test to ensure only users with the `can_view_users` permission can use this route?
Additionally if we allow users to look themselves up, could you add the scenarios where a user without `can_view_users` queries themselves successfully and where they query another user and get a 401?

Reply:
Done and done!